### DISCUSSION TITLE
Working on a new version

### DISCUSSION THREAD

**Discussion by barcoderdev** [barcoderdev/unitypackage_godot#27] [Show and tell]
![rewrite](https://github.com/barcoderdev/unitypackage_godot/assets/133794436/cf5d0604-f3cc-4b57-803c-1b210e72bef6)

Using godot-rust gdextension and completely rewritten rust and gdscript code, as an addon.

**Comment by AWRootbeard** [comment]
That's good, because the artifacts you built for the old version are all expired. 😅

**Comment by mikolop1** [comment]
When can we expect it?
